src/PresentationalComponents/CreateImageWizard/WizardStepImageOutput.js

```diff
@@ -1,40 +1,70 @@
+import { Form, FormGroup, FormSelect, FormSelectOption, Tile, Title } from '@patternfly/react-core';
+
+import './WizardStepImageOutput.scss';
 
 const WizardStepImageOutput = (props) => {
     const releaseOptions = [
         { value: 'rhel-8', label: 'Red Hat Enterprise Linux (RHEL) 8.3' },
     ];
-    const uploadOptions = [
-        { value: 'aws', label: 'Amazon Web Services' },
-    ];
+
     return (
-        <Form>
-            <Title headingLevel="h2" size="xl">Image output</Title>
-            <FormGroup isRequired label="Release" fieldId="release-select">
-                <FormSelect value={ props.value } onChange={ value => props.setRelease(value) } isRequired
-                    aria-label="Select release input" id="release-select" data-testid="release-select">
-                    { releaseOptions.map(option => <FormSelectOption key={ option.value } value={ option.value } label={ option.label } />) }
-                </FormSelect>
-            </FormGroup>
-            <FormGroup isRequired label="Target environment" fieldId="upload-destination">
-                <FormSelect value={ props.upload.type || '' } id="upload-destination"
-                    data-testid="upload-destination" isRequired
-                    onChange={ value => props.setUpload({ type: value, options: props.upload.options }) } aria-label="Select upload destination">
-                    { uploadOptions.map(option => <FormSelectOption key={ option.value } value={ option.value } label={ option.label } />) }
-                </FormSelect>
-            </FormGroup>
-        </Form>
+        <>
+            <Form>
+                <Title headingLevel="h2" size="xl">Image output</Title>
+                <FormGroup isRequired label="Release" fieldId="release-select">
+                    <FormSelect value={ props.value } onChange={ value => props.setRelease(value) } isRequired
+                        aria-label="Select release input" id="release-select" data-testid="release-select">
+                        { releaseOptions.map(option => <FormSelectOption key={ option.value } value={ option.value } label={ option.label } />) }
+                    </FormSelect>
+                </FormGroup>
+                <FormGroup isRequired label="Select target environment" fieldId="target-select">
+                    <div className="tiles">
+                        <Tile
+                            className="tile pf-u-mr-sm"
+                            title="Amazon Web Services"
+                            icon={ <img
+                                className='provider-icon'
+                                src={ '/apps/frontend-assets/partners-icons/aws.svg' } /> }
+                            onClick={ () => props.toggleUploadDestination('aws') }
+                            isSelected={ props.uploadDestinations.aws }
+                            isStacked
+                            isDisplayLarge />
+                        <Tile
```

Comment:
Should this be hidden until we have the support, or how will the experience be until we do?

Reply:
I can mark them as disabled for now so they still display but aren't selectable.

Reply:
I'd be good with that